app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```

Comment:
các file helper không có nội dung cũng xóa đi nhé, cần thì tạo.
Tư tưởng của công ty là tối ưu về line of code nghĩa là những cái trong pull chưa cần thiets thì xóa đi, sau này đều có thể tạo lại được